docs/language/overview.md

```diff
@@ -2025,35 +2000,35 @@ requires a type union for field `x` and produces:
 {x:2((int64,string)),y:"bar"}
 ```
 
-### 10.2 Fuse Function
+### 10.2 Fuse Aggregate Function
 
-The _fuse function_ is most often useful during data exploration and discovery
+The `fuse` aggregate function is most often useful during data exploration and discovery
 where you might interactively run queries to determine the shapes of some new
 or unknown input data and how those various shapes relate to one another.
 
-For example, in example sequence above, we can use the _fuse function_ to determine
+For example, in the example sequence above, we can use the `fuse` aggregate function to determine
 the fused type rather than transforming the values, e.g.,
 ```mdtest-command
-echo '{x:1} {x:"foo",y:"foo"}{x:2,y:"bar"}' | zq -z 'fuse(this)' -
+echo '{x:1} {x:"foo",y:"foo"} {x:2,y:"bar"}' | zq -z 'fuse(this)' -
 ```
 results in
 ```mdtest-output
 {fuse:<{x:(int64,string),y:string}>}
 ```
-Since the _fuse_ here is an aggregate function, it can also be used with
-group-by keys.  Supposing we wanted to fuse different type records into
+Since the `fuse` here is an aggregate function, it can also be used with
+group-by keys.  Supposing we wanted to fuse different categories of records into
 different types, we can use a group-by.  In this simple example, we will
-fuse records based on their number of fields using the `len()` function:
+fuse records based on their number of fields using the [`len` function:](functions/len.md)
```

Comment:
Please review this one closely. "...different type records into different types..." seemed tautological, so hopefully my change is ok.

Reply:
I think that's better but still not super clear. Maybe this?

```
Supposing we want to divide records into categories and fuse the records in each
category, we can use a group-by.
```

Reply:
@nwt: Sure, I like that wording. I've pushed a commit to use that wording and also adjusted the post-fix wrapping.